src/main/java/reactor/core/publisher/FluxSink.java

```diff
@@ -60,11 +62,27 @@
 
 	/**
 	 * Ensures that calls to next, error and complete are properly serialized.
+	 * {@link Flux#create(java.util.function.Consumer)} or
+	 * {@link Flux#create(java.util.function.Consumer, OverflowStrategy)} can be
+	 * used to create a serialized {@link FluxSink}.
 	 * @return the serialized {@link FluxSink}
 	 */
+	@Deprecated
 	FluxSink<T> serialize();
 
 	/**
+	 * Attaches a {@link LongConsumer} to this {@link FluxSink} that will be notified of
+	 * any request to this sink. This method is supported only on serialized sinks created
```

Comment:
Isn't this really recognizing there there should probably be a push-only `Sink` (used with `push`) and then something else that extends it and adds on `onRequest()` behaviour?  Having a method that is supported in one scenario, but not in another scenario isn't ideal.